pkg/cmd/accept/exec.go

```diff
@@ -70,10 +72,14 @@ func (o *Options) run() error {
 func (o *Options) runWithClient(kubeClient *kubernetes.Clientset, clusterClient *clusterclientset.Clientset) (err error) {
 	for _, clusterName := range o.values.clusters {
 		if o.wait == 0 {
-			_, err = o.accept(kubeClient, clusterClient, clusterName)
+			var csrApproved bool
+			csrApproved, err = o.accept(kubeClient, clusterClient, clusterName, false)
+			if err == nil && !csrApproved {
+				err = fmt.Errorf("no CSR to approve for cluster %s", clusterName)
```

Comment:
we did not check case with err != nil here...And the output message is here with line 150. You may want to add more details?

Reply:
yes we check err != nil at like 85.
The line 150 is displayed only when waitMode...  so it display continuesly until either the wait time is expired or the accept succeed.

Reply:
I see, thanks